lib/perl/Genome/Sys/Lock/FileBackend.pm

```diff
@@ -276,17 +277,11 @@ sub lock_dir_for_resource {
     return Path::Class::file($self->parent_dir, $resource_lock)->parent->stringify;
 }
 
-sub make_dir_path {
-    my ($self, $dir_path) = @_;
-    my $obj = Path::Class::dir($dir_path);
-    $obj->mkpath(0, 0777);
-}
-
 sub tempdir_for_resource {
     my ($self, $resource_lock) = @_;
 
     my $lock_dir = $self->lock_dir_for_resource($resource_lock);
-    $self->make_dir_path($lock_dir);
+    Genome::Sys->create_directory($lock_dir . '');
```

Comment:
My best guess is that this is here to force `$lock_dir` to stringify, but it [appears](https://github.com/nnutter/genome/blob/lock-pair/lib/perl/Genome/Sys/Lock/FileBackend.pm#L277) that we already have a string. Am I missing something? (Highly likely!). If it is to force stringfication, is there a reason to prefer this to `"$lock_dir"`?